src/components/pager/pager.styles.js

```diff
@@ -13,12 +9,12 @@ const PagerContainerStyles = styled.div`
   padding: 9px 24px;
   align-items: center;
   border-top-width: 0;
-  font-size: 13px;  
+  font-size: 13px;
   ${({ theme }) => {
     return theme.table && theme.colors && css`
-      border: 1px solid ${theme.table.selected};
+      border: 1px solid #CCD6DA;
```

Comment:
we shouldn't have hardcoded colors in our styles

Reply:
fixed 4a893a7